src/libs/ReportActionsUtils.ts

```diff
@@ -1348,6 +1350,34 @@ function isLinkedTransactionHeld(reportActionID: string, reportID: string): bool
     return TransactionUtils.isOnHoldByTransactionID(getLinkedTransactionID(reportActionID, reportID) ?? '-1');
 }
 
+function getAccountIDsFromMessage(message: string) {
+    const regex = /<mention-user\s+accountID="(\d+)"\/>/g;
```

Comment:
I think we should rename this variable to make it more understandable.
```suggestion
    const accountIDRegex = /<mention-user\s+accountID="(\d+)"\/>/g;
```